### PR TITLE
Added description to introduce cssls

### PR DESCRIPTION
**Pull request by nabezokodaikon**

### CODE REVIEW

File: lua/lspconfig/cssls.lua [hunk 1]
```diff
@@ -27,6 +27,19 @@ https://github.com/vscode-langservers/vscode-css-languageserver-bin
 ```sh
 npm install -g vscode-css-languageserver-bin
 ```
+
+Neovim does not currently include built-in snippets. `vscode-html-languageserver` only provides completions when snippet support is enabled.
```
Comment (mjlbach) [inline lua/lspconfig/cssls.lua:31]: Still need to fix this type :) vscode-html-languageserver -> vscode-css-languageserver

**Comment by mjlbach** [comment]
Looks good, now you just need to squash.

**Comment by cloggier** [comment]
The same applies to other lsps so maybe extract it into separate section and link to from relevant servers?

**Comment by mjlbach** [comment]
We can't do that, I'd have to rewrite the docgen (and I won't)

**Comment by nabezokodaikon** [comment]
Is my PR method correct?
I feel bad. I am not used to PR.

**Comment by mjlbach** [comment]
It's great :) Thanks for your contribution.

**Pull request merged**